index.html

```diff
@@ -192,6 +192,33 @@ <h3 class="head">🔸Skill</h3>
                         
                     </div>
                 </section>
-               
+           <section class="section_3">
+            <h2 class="description-2">I'm always interested in hearing about new projects,
+                       so if you'd like to chat please get in touch.</h2>
+                       <form action="">
```

Comment:
- [ ] You did a good job wrapping all the `<submit>` elements inside `<form>` :+1:, but as a project requirement you should add `action` and `method` attributes to `<form>` and collect all data by using the [Formspree service](https://formspree.io/html) :100: